test/unit-test/ota_utest.c

```diff
@@ -154,6 +160,23 @@ extern OtaErr_t jobNotificationHandler( const OtaEventData_t * pEventData );
 extern OtaErr_t setImageStateWithReason( OtaImageState_t stateToSet,
                                          uint32_t reasonToSet );
 
+/* Static helper function to help initialize the Json document model. */
```

Comment:
May want  to remove these one line comments and make them more flush like it is with the state machine handler functions.  I would just change the "/*Static helper function under test defined in ota.c. */" to be plural. This way would take up less space and maintain a single source of truth for the description of what the function does.

Reply:
Done